src/components/slices/intro-description.module.scss

```diff
@@ -0,0 +1,96 @@
+@import '../../base/module';
+
+.introBox {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  padding: 110px 0;
+  background-color: $color-dove;
+  position: relative;
+  margin-bottom: 10px;
+}
+
+.palm::before {
+  content: '';
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: -10px;
+  z-index: 1;
+  width: 360px;
+  height: 190px;
+  background-color: $color-sage;
+  mask: url('../../images/palm.svg') no-repeat 100% 100%;
+  mask-size: contain;
+  transform: rotate(-10deg) translateX(-50%) translateY(10%);
+  @media (min-width: $desktop) {
+    transform: rotate(-10deg) translateX(-15%) translateY(30%);
+    width: 560px;
+    height: 390px;
+  }
+}
+
+.textBox {
+  width: 85%;
+  text-align: left;
+  margin-bottom: 30px;
+  p {
+    margin-bottom: 30px;
+  }
+  @media (min-width: $tablet) {
+    width: 60%;
+  }
+}
+
+.titleText {
+  text-align: center;
+  font-family: $font-secondary;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 600;
```

Comment:
Yeah the font weight here is getting killed because of the p tag. These styles will work better once it's an H tag :)